### ISSUE TITLE
Problem With Layout/HashAlignment

### ISSUE THREAD

**Issue by jherdman** [rubocop/rubocop#9871]
## Expected behavior

The Layout/HasAlignment cop works.

## Actual behavior

```
Scanning /home/james/DEV/PN/eternal-sledgehammer/app/controllers/subscriptions_controller.rb
An error occurred while Layout/HashAlignment cop was inspecting /home/james/DEV/PN/eternal-sledgehammer/app/controllers/subscriptions_controller.rb:96:28.
undefined method `loc' for nil:NilClass
/home/james/DEV/PN/eternal-sledgehammer/vendor/cache/ruby/2.6.0/gems/rubocop-1.16.1/lib/rubocop/cop/layout/hash_alignment.rb:221:in `autocorrect_incompatible_with_other_cops?'
/home/james/DEV/PN/eternal-sledgehammer/vendor/cache/ruby/2.6.0/gems/rubocop-1.16.1/lib/rubocop/cop/layout/hash_alignment.rb:206:in `on_hash'
/home/james/DEV/PN/eternal-sledgehammer/vendor/cache/ruby/2.6.0/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:100:in `public_send'
```

Offending code:

```ruby
params
  .some_method
  .fetch(:foo, {})
  .more
  .another
```

We see an _enormous_ number of these problems. This cop has no such issues using 1.16.0.

## Steps to reproduce the problem

Simply run the cop. 

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
1.16.1 (using Parser 3.0.1.1, rubocop-ast 1.7.0, running on ruby 2.6.6 x86_64-linux)
  - rubocop-performance 1.11.3
  - rubocop-rails 2.10.1
  - rubocop-rspec 2.4.0
```

**Comment by dvandersluis** [comment]
This was fixed in #9862 and will be released soon. Thanks!
